src/BenchmarksDriver/Program.cs

```diff
@@ -508,16 +519,17 @@ public static int Main(string[] args)
                 }
                 if (pipelineDepthOption.HasValue())
                 {
-                    _clientJob.PipelineDepth = int.Parse(pipelineDepthOption.Value());
+                    var pipelineDepth = int.Parse(pipelineDepthOption.Value());
+                    _clientJob.WorkerProperties["PipelineDepth"] = pipelineDepth;
```

Comment:
This could be handled in a generic way were we pass client specific arguments to the worker properties.
Something like `--client wrk --properties thread=16;connections=32;pipelinedepth=16` or similar

Reply:
Do we want to get rid of the `--clientThreads` `--script` and `--pipelineDepth` options